ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -94,6 +94,7 @@ do
   cp "${item}" "${dir}/tls/${manifest##tls_}"
 done <   <( find "${SHARED_DIR}" \( -name "tls_*.key" -o -name "tls_*.pub" \) -print0)
 
+echo "$(date +%s)" > "${SHARED_DIR}/CLUSTER_INSTALL_START_TIME"
```

Comment:
The format for this timestamp is not friendly for human consumption, which is the primary purpose of the timestamp.
```
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/17804/rehearse-17804-pull-ci-openshift-installer-release-4.9-e2e-aws-shared-vpc/1385774306436321280/artifacts/e2e-aws-shared-vpc/gather-extra/artifacts/configmaps.json | jq '.items[] | select(.metadata.namespace=="openshift-config") | select(.metadata.name=="openshift-install") | .data'
{
  "endTime": "1619231956",
  "invoker": "openshift-internal-ci/rehearse-17804-pull-ci-openshift-installer-release-4.9-e2e-aws-shared-vpc/1385774306436321280",
  "startTime": "1619230238",
  "version": "unreleased-master-4511-g0ad3eefc546ca394ca0de61582610a0f30f6433a-dirty"
}
```